uint8 box_count
bbox_msg[] bbox_array
sensor_msgs/Image img_narrow